frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -61,6 +61,26 @@ function getDesignSystemCssVariables(theme: MantineTheme) {
       var(--mb-color-brand) 88%,
       transparent
     );
+    --mb-color-border-alpha-30: color-mix(
```

Comment:
Add colors that [are mistakenly removed](https://github.com/metabase/metabase/commit/3d5233bc6a18135766b2f06e741929c4e6621d8f#diff-f11ce598e3fb0b9484b4e5418d6befa090a84fb411dd1d0b93c00bfe4dced509L48-L67).

Since different colors are defined in different files, I mistakenly removed 4 of them.

This should fix some modal backdrop color.